packages/react-core/src/components/Wizard/WizardNavItem.tsx

```diff
@@ -49,16 +53,36 @@ export const WizardNavItem: React.FunctionComponent<WizardNavItemProps> = ({
   };
 
   return (
-    <li className={css(styles.wizardNavItem)}>
+    <li
+      className={css(
+        styles.wizardNavItem,
+        isExpandable && styles.modifiers.expandable,
+        isExpandable && isCurrent && styles.modifiers.expanded
+      )}
+    >
       <NavItemComponent
         {...rest}
         {...(navItemComponent === 'a' ? { ...linkProps } : { ...btnProps })}
         onClick={() => onNavItemClick(step)}
-        className={css(styles.wizardNavLink, isCurrent && 'pf-m-current', isDisabled && 'pf-m-disabled')}
+        className={css(
+          styles.wizardNavLink,
+          isCurrent && styles.modifiers.current,
+          isDisabled && styles.modifiers.disabled
+        )}
         aria-disabled={isDisabled ? true : null}
         aria-current={isCurrent && !children ? 'page' : false}
+        {...(isExpandable && { 'aria-expanded': isCurrent })}
       >
-        {content}
+        {isExpandable ? (
+          <>
+            <span className={css(styles.wizardNavLinkText)}>{content}</span>
+            <span className={css(styles.wizardNavLinkToggleIcon)}>
```

Comment:
I found an issue here and [put up a PR](https://github.com/patternfly/patternfly/pull/4112) I'd like to merge tomorrow, it will create `<span className={css(styles.wizardNavLinkToggle)} />` that will wrap this icon element.

Reply:
@nicolethoen the core PR was merged so we can go ahead and add that element in this PR.